src/PowerShellEditorServices/Utility/Utils.cs

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System.Runtime.InteropServices;
```

Comment:
For convenience if explicit culture is added: 

```suggestion
using System;
using System.Runtime.InteropServices;
```